packages/typography/README.md

```diff
@@ -0,0 +1,70 @@
+# Typography
+
+![npm (scoped)](https://img.shields.io/npm/v/@leafygreen-ui/typography.svg)
+
+## Installation
+
+### Yarn
+
+```shell
+yarn add @leafygreen-ui/typography
+```
+
+### NPM
+
+```shell
+npm install @leafygreen-ui/typography
+```
+
+# H1
+
+## Example
+
+```js
+import { H1, H2, Subtitle, Body, InlineCode, Disclaimer, Overline } from '@leafygreen-ui/typography';
+<H1>Heading 1</H1>
```

Comment:
The syntax highlighting for this is turning out strange. Could you try `jsx` instead of `js` for the language?

Reply:
yes